src/app/create-user/create-user.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { UserModel } from '../user.model';
+import { UsersService } from '../users.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-create-user',
+  templateUrl: './create-user.component.html',
+  styleUrls: ['./create-user.component.css']
+})
+export class CreateUserComponent implements OnInit {
+
+  constructor(private usersService: UsersService, private route: Router) { }
+
+  ngOnInit(): void {
+  }
+
+  handleNewUser(userData: UserModel): void {
+    this.usersService.createUser(userData).subscribe(
+      (result)=> result,
+      (error) => console.log("Error at handleNewUser ", error),
+      () => this.route.navigate(['manage'])
```

Comment:
Can we navigate the user only on success of the request ? Right now if there is an error as well, it navigates back.